src/Token/AbstractToken.php

```diff
@@ -181,10 +181,8 @@ public function getToken()
 
     /**
      * Generic accessor to enable access as public properties.
-     *
-     * @return string
      */
-    public function __get($key)
+    public function __get(string $key): mixed
```

Comment:
The `mixed` return type is OK here, but the `string` parameter type is a BC break